be/src/storage/memtable_flush_executor.cpp

```diff
@@ -83,20 +94,18 @@ void FlushToken::_flush_memtable(std::shared_ptr<vectorized::MemTable> memtable)
 }
 
 Status MemTableFlushExecutor::init(const std::vector<DataDir*>& data_dirs) {
-    int32_t data_dir_num = data_dirs.size();
-    size_t min_threads = std::max(1, config::flush_thread_num_per_store);
-    size_t max_threads = data_dir_num * min_threads;
+    int data_dir_num = static_cast<int>(data_dirs.size());
+    int min_threads = std::max<int>(1, config::flush_thread_num_per_store);
+    int max_threads = data_dir_num * min_threads;
     return ThreadPoolBuilder("MemTableFlushThreadPool")
             .set_min_threads(min_threads)
             .set_max_threads(max_threads)
             .build(&_flush_pool);
 }
 
-// NOTE: we use SERIAL mode here to ensure all mem-tables from one tablet are flushed in order.
-OLAPStatus MemTableFlushExecutor::create_flush_token(std::unique_ptr<FlushToken>* flush_token,
-                                                     ThreadPool::ExecutionMode execution_mode) {
-    *flush_token = std::make_unique<FlushToken>(_flush_pool->new_token(execution_mode));
-    return OLAP_SUCCESS;
+std::unique_ptr<FlushToken> MemTableFlushExecutor::create_flush_token(ThreadPool::ExecutionMode execution_mode) {
+    SCOPED_THREAD_LOCAL_MEM_TRACKER_SETTER(nullptr);
```

Comment:
Better to add some comment why change mem tracker in this function.

Reply:
It's unused now, I removed it.